docs/mods.md

```diff
@@ -17,9 +17,9 @@
-|                                          [Immersive Armors](https://www.curseforge.com/minecraft/mc-mods/immersive-armors)                                           | <u>Mod Versions</u><br>All of MC `1.16.5`<br>All of MC `1.17.1`<br>All of MC `1.18.1`<br>All of MC `1.19`<br>All of MC `1.19.1`<br><br><u>Mod Versions Below</u><br>**1.18.2**-`1.5.6`<br>**1.19.2**-`1.5.6`<br>**1.19.3**-`1.5.6`<br>**1.19.4**-`1.5.6`<br>**1.20.1**-`1.5.6` | <u>Mod Versions and Up</u><br>**1.18.2**-`1.5.6`+<br>**1.19.2**-`1.5.6`+<br>**1.19.3**-`1.5.6`+<br>**1.19.4**-`1.5.6`+<br>**1.20.1**-`1.5.6`+ | ✅ <br>[Project Patch](https://github.com/dogboy21/serializationisbad/blob/871357f6763c59824b84b151d076d537f7345b8c/serializationisbad.json#L61-L69) |                                                                                          |                                                                                                                    |
-|                                        [Immersive Aircraft](https://www.curseforge.com/minecraft/mc-mods/immersive-aircraft)                                         |                                                                                                                                      All                                                                                                                                       |                                                                     None                                                                      |                                                            ❌ <br> Patch Broken Currently                                                            |                                                                                          |                                                                                                                    |
-|                                       [Immersive Paintings](https://www.curseforge.com/minecraft/mc-mods/immersive-paintings)                                        |                                                                                                                                      All                                                                                                                                       |                                                                     None                                                                      |                 ✅ <br> [Project Patch](https://github.com/dogboy21/serializationisbad/blob/master/serializationisbad.json#L79-L87)                  |                                                                                          |                                                                                                                    |
```

Comment:
Looks like you accidentally removed the project patch for Immersive Paintings. Think you could re-add that back?

Reply:
Whoops, added it back in.